docs/API_SPECS_SIMULATIONS.md

```diff
@@ -466,35 +472,55 @@ Content-Type: application/json; charset=utf-8
 ```
 ```json
 {
-  "ETag": "cec0722b205740",
+  "ETag": "969ee1fb277640",
   "Id": "1",
-  "Enabled": true,
-  "StartTime": "2019-02-03T14:00:00",
-  "EndTime": "2019-02-04T00:00:00",
+  "Name": "Sample Simulation",
+  "Description": "This is a sample simulation",
+  "Enabled": false,
+  "Running": false,
+  "IoTHubs": [
+      {
+          "ConnectionString": "default",
+          "PreprovisionedIoTHub": true,
+          "PreprovisionedIoTHubInUse": false,
+          "PreprovisionedIoTHubMetricsUrl": null
```

Comment:
these 2 seems inconsistent. If the simulation is using the `default` connection string, then `PreprovisionedIoTHubInUse` should be true, and `PreprovisionedIoTHubInUse` should have a value